primefaces/src/main/resources/META-INF/resources/primefaces/validation/validation.common.js

```diff
@@ -441,15 +482,13 @@ if (window.PrimeFaces) {
                 element.attr('aria-invalid', false);
             }
             else {
-                if (highlight) {
-                    highlighter.highlight(element);
-                }
+                highlighter.highlight(element, !highlight);
```

Comment:
.highlight is called with the second param, but .unhighlight not, right?

Reply:
Yep. "unhighlight" does essentially the same work as before. When we call highlight from the new codepaths we don´t want  to show e.g. messages, so we need the additional parameter.

Reply:
should we just pass "false" then? everything else looks weird :D

Reply:
ok, i´ll add it in den evening.